src/components/audio-control-bar/audio-control-bar.test.tsx

```diff
@@ -1,12 +1,18 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { renderWithTestRoots } from '@/app.test';
 import { noop } from '@/helpers/func';
 import { AudioControlBar } from './audio-control-bar';
 
 describe('StudyFlashcard', () => {
```

Comment:
I think I forgot to update the test suite name. Could you update that for me?

Reply:
done! also added a test for the volume slider